src/app/settings-modal.tsx

```diff
@@ -1,102 +0,0 @@
-export default function SettingsModal({
-  settings,
-  handleChange,
-  toggleModal,
-}: any) {
-  return (
-    <>
-      {/* ----->  Taken from Tailwind CSS example: https://tailwindui.com/components/application-ui/overlays/dialogs */}
-      <div
-        className="relative z-10"
-        aria-labelledby="modal-title"
-        role="dialog"
-        aria-modal="true"
-      ></div>
-      <div className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"></div>
```

Comment:
If we want the background to be grayed out while the modal is open, I believe this is the line that accomplished that for me from the TailwindCSS example. I think it looks fine as it is now though